frontend/components/ErrorNotFoundComponent.js

```diff
@@ -1,18 +0,0 @@
-import React from "react";
```

Comment:
Is there a reason this file was deleted? We should probably keep it.

Reply:
I am pretty sure it got added automatically when I accidentally deleted a component I was working on. It caused a bunch of errors and reformatted some code until I added the original file back and deleted this one.